### PR TITLE
A  real chat should be realtime 💬 🚀

### PR DESCRIPTION
**Pull request by kvartborg**

### CODE REVIEW

File: src/views/Task/Chat.js [hunk 1]
```diff
@@ -20,6 +21,19 @@ export default class Chat extends Component {
 
   componentDidMount() {
```
Comment (grammeaway) [inline src/views/Task/Chat.js]: Docblock

File: src/views/Task/Chat.js [hunk 2]
```diff
@@ -20,6 +21,19 @@ export default class Chat extends Component {
...
     this.loadMessages(this.props.taskId);
+    this.props.broker.on(`message/${this.props.taskId}`, this.listenForMessages)
+  }
+
+  componentWillUnmount() {
```
Comment (grammeaway) [inline src/views/Task/Chat.js]: Docblock

Comment (kvartborg) [reply src/views/Task/Chat.js]: fixed a648b6c

File: src/views/Task/Chat.js (hunk 1 above)
Comment (kvartborg) [reply src/views/Task/Chat.js]: fixed a648b6c

**Pull request merged**